pkgs/by-name/mi/mingtest/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  stdenv,
+  cmake,
+  texliveSmall,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation {
+  name = "mingtest";
+  version = "0.1.9-4";
+  src = fetchFromGitHub {
+    owner = "craflin";
+    repo = "mingtest";
+    rev = "b6af8ada0908ea855a563ef43d5543bab8512d7d";
+    sha256 = "PSJ1+PtPR59qO/msCTS3o2ShjYkhO4h0Mtn0XY+KluQ=";
+  };
+
+  postPatch = ''
+    sed -i -e 's$include(CDeploy)$$' CMakeLists.txt
+    sed -i -e 's$install_deploy_export()$$' CMakeLists.txt
+  '';
+
+  buildInputs = [
+    cmake
```

Comment:
This should be in `nativeBuildInputs`

Reply:
@Aleksanaa Is it similar to hooks? 
Something like if you contains that it will override the buildPhase ?
( I haven't found anything about it in https://github.com/NixOS/nixpkgs/blob/master/doc/hooks/cmake.section.md )

Reply:
I'm a bit confused about @jackyliu16 comment. So should it be cmake defintitly in nativBuildInputs?

Reply:
Please move it to `nativeBuildInputs`.
If you want to know more, see https://nixos.org/manual/nixpkgs/unstable/#ssec-stdenv-dependencies